fragment-rigger/src/main/java/com/jkb/fragment/rigger/annotation/Puppet.java

```diff
@@ -54,4 +54,15 @@
    * @return The default value is false.
    */
   boolean stickyStack() default false;
+  
+  /**
+   * Allow or not exit the host {@link android.app.Activity}/{@link android.support.v4.app.Fragment} when
+   * the stack size = 0
+   *
+   * If the value is true , the host object(Activity/Fragment) will not be exit
+   * Otherwise , the host object(Activity/Fragment) will act as normal.
+   *
+   * @return The default value is true.
+   */
+  boolean closeIfEmpty() default true;
```

Comment:
did you tried `bondContainerView = true` or  `stickyStack = true` ?

Reply:
yes i was but it just not work on my case.
Example: i have main screen with 5 fragments (5 nav menu items) and i using `addFragment` to add it and this method do not add to stack. so when i create new fragment with `startFragment` and after finish i close it. it will check `onBackPress` because my stack = 0 so it close activity

Reply:
@JustKiddingBaby check `show` demo you will see it close application when you press back button

Reply:
@namkazt OK, i got it , thanks for the PR , i will review in weekend and merge it as soon .

Reply:
@namkazt  please change the target branch to dev

Reply:
okey. will fix conflict when have free time.